DEVELOPMENT.md

```diff
@@ -1,13 +1,16 @@
 Development
 ===========
 
+We use [builder][] and `npm` to control all aspects of development and
+publishing.
+
 ## Build
 
 Build for production use (NPM, bower, etc) and create `dist` UMD bundles
 (min'ed, non-min'ed)
 
 ```
-$ npm run build
+$ builder run build
```

Comment:
It might be worthwhile to have a `cli` tool that checks for a project's instance of `builder` so people aren't having to modify their paths. I've found that to be the easiest way to maintain consistency with end users, especially since people are used to installing things globally.